app/dojo.py

```diff
@@ -0,0 +1,141 @@
+from room import LivingSpace, Office
+from person import Fellow, Staff
+from random import choice
+
+class Dojo:
+	"""
+	A class that contains the methods to create a new room and to add new people to the 
+	Dojo. 
+
+	"""
+
+	def __init__(self):
+
+		self.living_spaces = {"Amugune": ["Jonathan Swift", "Tom Jones"]}
+		self.offices = {"Saitoti": { "Fellows":["Jonathan Swift", "Tom Jones"], "Staff":[]}, "Sahari": {"Staff":["William Jakoyo", "Franklin Barasa"], "Fellows":[]} }
+		self.fellows = ["Jonathan Swift", "Tom Jones"]
+		self.staff = ["William Jakoyo", "Franklin Barasa"]
+
+	def create_room(self, room_type, room_name_list):
+		"""Create a new room."""
+
+		self.room_type = room_type
+
+		if self.room_type not in ["Office", "Living-space"]:
+			print("The Dojo doesn't have the room type you've entred")
+
+		else:
+			
+			for room_name in room_name_list:
+				self.room_name = room_name
+
+				"""Check whether the room already exists"""
+				if self.room_name in self.offices or self.room_name in self.living_spaces:
```

Comment:
Thanks a lot. This was my biggest challenge: accessing an object in a list. It was why i kept changing them into strings.